lib/sentry/client.ex

```diff
@@ -257,14 +276,14 @@ defmodule Sentry.Client do
     result
   end
 
-  @spec maybe_call_before_send_event(Event.t()) :: Event.t()
+  @spec maybe_call_before_send_event(Event.t()) :: Event.t() | false
```

Comment:
Why false instead of :error or something more descriptive.

Reply:
this kind of wraps the user-implemented callback, so there shouldn't be an error case. The user's callback should return an event or something falsey. If it's an event, it means Sentry will send as normal, but if the callback returns something false-y, this function will return `false` (and we won't send).